src/test/java/duke/command/AddCommandTest.java

```diff
@@ -0,0 +1,26 @@
+package duke.command;
+
+import duke.main.Ui;
+import duke.utils.Storage;
+import duke.utils.StorageStub;
+import duke.utils.TaskList;
```

Comment:
You packaged your files in a very orderly manner, a different package for different functionalities, nice work!